misc/FFNx.cfg

```diff
@@ -15,46 +15,73 @@
 # COMMON OPTIONS FOR BOTH FF7 AND FF8
 ###############################################################################
 
-## RENDERING
+#########################
+## DISPLAY
+#########################
 
+#[RENDERING BACKEND]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Available choices are:
-### SUGGESTED:
 # - OpenGL ( default )
 # - Direct3D11 ( valid alternative for OpenGL)
-### NOT SUGGESTED (TECHNICAL POC):
-# - Vulkan ( slow performance)
-# - Direct3D12 ( slow performance )
+# - Vulkan ( Tech Preview: Not Recommened )
+# - Direct3D12 ( Tech Preview: Not Recommened )
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #renderer_backend = OpenGL
 
-# Enable fullscreen mode.
+#[FULLSCREEN]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # If off, it will run in window mode.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #fullscreen = no
 
+#[RESOLUTION]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Resolution of the game.
-# If any of the two is 0, the driver will use original game resolution ( 640x480 ) in window mode.
-# Your desktop resolution in fullscreen mode otherwise.
+# Default (value = 0):
+# - Window mode will use 640x480
+# - Fullscreen mode will use your desktop resolution
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #window_size_x = 0
 #window_size_y = 0
 
+#[INTERNAL RESOLUTION SCALE]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Internal Scale used it Perserve Aspect is not enabled.
 # Internal game resolution scale.
-# By default is set to four times the calculated game rendering resolution, that is nearest to 640x480, if output resolution is not within the 4:3 aspect ratio.
+# By default is set to four times the calculated game rendering resolution, that is nearest to 640x480
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #internal_resolution_scale = 4
 
-# Preserve original game aspect ratio (4:3) by adding black bars on the left and right side.
-# If you wish to have it fully stretched, keep this option off. Be aware the game may look wrong though.
+#[PRESERVE ASPECT]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Preserve original game aspect ratio of (4:3) by adding black bars on the left and right side (if needed)
+# When off the game will be stretched to fit the windows aspect ratio; Be aware the game may look wrong though.

```

Comment:
windows -> window

Reply:
There's still `windows` instead of `window` :P

Reply:
should be window's .. 
Must have missed this in my update